File: examples/python/plot2table_example.py
import qoery

configuration = qoery.Configuration()
configuration.api_key['ApiKeyAuth'] = 'YOUR_API_KEY'

with qoery.ApiClient(configuration) as api_client:
    api = qoery.WebScrapingApi(api_client)
    data = api.plot2table_get(image_url="https://example.com/chart.png")
    print(data)

File: examples/js/plot2table_example.js
// Example: Convert chart image to data table via SDK (GET)
const Qoery = require('qoery');

const client = Qoery.ApiClient.instance;
const apiKeyAuth = client.authentications['ApiKeyAuth'];
apiKeyAuth.apiKey = 'YOUR_API_KEY';

const scrapingApi = new Qoery.WebScrapingApi();

const opts = { imageUrl: 'https://example.com/chart.png' };

scrapingApi.plot2tableGet(opts, (error, data) => {
  if (error) {
    throw error;
  }
  console.log(data);
});
